src/ir_def/arrange_ir.ml

```diff
@@ -26,7 +26,7 @@ let rec exp e = match e.it with
   | AsyncE (Type.Cmp, tb, e, t) -> "AsyncE*"  $$ [typ_bind tb; exp e; typ t]
   | DeclareE (i, t, e1) -> "DeclareE" $$ [id i; exp e1]
   | DefineE (i, m, e1)  -> "DefineE" $$ [id i; mut m; exp e1]
-  | FuncE (x, s, c, tp, as_, ts, e) ->
+  | FuncE (x, s, c, tp, as_, ts, _, e) ->
```

Comment:
Might be worth displaying the closure environment

Reply:
Agree.